frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -152,11 +157,25 @@ function Pages(): JSX.Element {
                         identifier={Scene.SavedInsights}
                         to={urls.savedInsights()}
                         sideAction={{
-                            icon: <IconPlus />,
```

Comment:
I think it's better to keep the plus button even if the _standard_ pattern is to use the arrow for dropdowns. The plus is just such a clear affordance here